Source/WebCore/html/HTMLFormElement.cpp

```diff
@@ -195,13 +195,14 @@ void HTMLFormElement::submitImplicitly(Event& event, bool fromImplicitSubmission
 {
     unsigned submissionTriggerCount = 0;
     for (auto& listedElement : m_listedElements) {
-        if (!is<HTMLFormControlElement>(*listedElement))
+        auto* formElement = dynamicDowncast<HTMLFormControlElement>(*listedElement);
+        if (!formElement)
             continue;
```

Comment:
I have a slight preference for wrapping the whole block with
`if (auto* formElement = dynamicDowncast<HTMLFormControlElement>(*listedElement))` rather than early `continue;` here.

Reply:
In this instance, I prefer to keep it this way as it matches more the WebKit coding style where we try to early return to minimize nesting.

I often make an exception for one-liners but here the body is larger than that.